packages/SwingSet/src/kernel/vat-loader/manager-factory.js

```diff
@@ -54,66 +53,67 @@ export function makeVatManagerFactory({
     !setup || enableSetup || Fail`setup() provided, but not enabled`;
   }
 
-  // returns promise for new vatManager
-  function vatManagerFactory(
+  /**
+   * returns promise for new vatManager
```

Comment:
```suggestion
   * Asynchronously creates a manager for a vat with specified vatID, options,
   * and a provided vat syscall handler function.
```
Also, should this `vatManagerFactory` function be renamed to e.g. `makeVatManager`?

Reply:
kind of? That's what factories are/do, so I think they're both sensible names. In this case `vatManagerFactory` delegates off to two other sub-factories depending upon the `workerOptions.type` (which used to be `managerType`). The name `makeVatManager` doesn't capture what kind types it handles, though: changing this one to `makeVatManager` would really want the sub-factories to be named like `makeLocalVatManager` and `makeXsnapVatManager`, and that seems like work without a big clarity reward.